src/main/java/com/tg5/service/LowBalanceCronAlert.java

```diff
@@ -0,0 +1,45 @@
+package com.tg5.service;
+
+import com.tg5.domain.Account;
+import com.tg5.domain.AccountType;
+import com.tg5.domain.Member;
+import com.tg5.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class LowBalanceCronAlert {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    @Autowired
+    private MessageProducer messageProducer;
+
+    @Scheduled(fixedRate = 50000) // Execute every 5 seconds
+    public void sendLowBalanceAlertMessage() {
+        System.out.println("Checking all low balance accounts and sending mails.......");
+//        messageProducer.sendMessage("ams",
```

Comment:
if this is unnecessary testing you can remove it

Reply:
Discussed it: It is just for demo!